Curs/.stylelintrc

```diff
@@ -1,6 +1,6 @@
 {
-    "extends": "stylelint-config-standard",
+    "extends": "stylelint-config-standardsdaaada",
```

Comment:
I have a feeling that changes to this file are unintentional and thus should be removed.

Reply:
Something happened there. :)

Is done now.